### PR TITLE
Adjustments for JDK 11

### PR DESCRIPTION
**Pull request by goldmann**
* Renamed the OpenJDK 11 product image to `rhoar-openjdk/openjdk-11-rhel7`.
* Removed the Fedora version of the image
* Adjusted readme and build descriptor

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
 ```
 
 This will produce an image named `redhat-openjdk-8/openjdk18-openshift`
 
-#### JDK 11
+### RHEL JDK 11
+
 ```
-cekit --overrides-file=product-overrides.yaml --overrides-file=rhel7-jdk11-overrides.yaml build
+cekit --overrides-file=rhel7-jdk8-overrides.yaml --overrides-file=rhel7-jdk11-overrides.yaml build
```
Comment (rcernich) [inline README.adoc:40]: should we add `--redhat` flag to this?

**Review by rcernich — DISMISSED** [review]
LGTM.

Do we want to use overrides for the different jdk versions, or separate image.yaml files?  Does it matter?

File: README.adoc (hunk 1 above)
Comment (goldmann) [reply README.adoc:40]: I don't think so. This enables features specific to Red Hat environment, but should not really influence the image itself.

**Review by jmtd — DISMISSED** [review]
LGTM pending rhoar

**Review by jmtd — APPROVED** [review]
LGTM

**Pull request merged**